.rubocop.yml

```diff
@@ -49,6 +49,7 @@ Metrics/BlockLength:
     - resource
     - context
     - describe
+    - let
```

Comment:
I wouldn't include `let` here. I would have an inline exception for this (or include it in `.rubocop_todo.yml`), with a view to trying to fix it in the future.

Reply:
we are ignoring pretty much everything else for rspec though, including `it` and `specify`, which are arguably more important to restrict the size of -  i mention this in commit, so am not sure why we would not just add this too?!